livekit-agents/livekit/agents/voice/room_io.py

```diff
@@ -71,12 +71,13 @@ def __init__(
         link_to_participant: Optional[rtc.RemoteParticipant | str] = None,
         input_options: RoomInputOptions = DEFAULT_ROOM_INPUT_OPTIONS,
         output_options: RoomOutputOptions = DEFAULT_ROOM_OUTPUT_OPTIONS,
+        noise_cancellation: rtc.NoiseCancellationOptions | None = None,
```

Comment:
Can you add this to the RoomInputOptions, this way we can directly use it inside the VoiceAgent without having to manually create the RoomIO. wdyt?

Reply:
the start method of VoiceAgent has a RoomInputOptions